popSimManu.tex

```diff
@@ -163,10 +172,15 @@ \section*{Introduction}
 %ADK: made some edits here
 %RNG: Removed the sap. :-P Feel free to put it back if you like it better.
 %JK: not crazy about it either. Making it shorter helps though?
+%ALG: I think it sounds fine.
+%ALG: Do we also need to say something about how it’s useful to people not in methods development? 
```

Comment:
i think a lot of these comments are dated at this point... and @asiepel is about to go a bit of an edit right here